docs/cluster_and_device_type_dev/cluster_and_device_type_dev.md

```diff
@@ -172,10 +172,10 @@ persistence
 
 -   When using AttributeAccessInterface, you need to manage any Attributes that
     require Persistence.
--   This can be done by using GetSafeAttributePersistenceProvider()
+-   This can be done by using GetAttributePersistenceProvider()
```

Comment:
I would mention that injecting this is often suggested for unit testing.

Point people to use `TestPersistentStorageDelegate` with `DefaultAttributePersistenceProvide` for unit testing.